### PR TITLE
Add Login Form Component with Validation and User State

### PR DESCRIPTION
**Pull request by ShanigarapuVinay**
### Summary
This PR introduces a new `Form` component that handles user login functionality. The component includes basic validation for the username and password fields and manages the user login state.

### Changes
- Added a new `Form` component.
- Implemented `useState` hooks to manage `username`, `password`, and `isLoggedIn` state.
- Included basic form validation to ensure that both username and password are not empty.
- Added `useEffect` hook to log a message when the component mounts.
- Displayed user details upon successful login.
- Added appropriate alert messages for validation and login success.
- Applied basic styling using CSS classes `form`, `home-container`, and `form-container`.

### Screenshots

image

image

image

image

image

### CODE REVIEW

File: Form.jsx
Comment (AnnaHariprasad5123) [inline Form.jsx:1]: You should use arrow function and follow next time.

Comment (AnnaHariprasad5123) [inline Form.jsx:1]: You should use arrow function and follow next time.

**Review by AnnaHariprasad5123 — APPROVED** [review]
Looks good to me.

File: Form.jsx
Comment (ShanigarapuVinay) [reply Form.jsx:1]: Yeah sure

**Pull request merged**